config.yml

```diff
@@ -3,3 +3,8 @@ email:
     notify:
         - "rjewing@bu.edu"
         - "dlhi@bu.edu"
+scheduler:
+    alarm:
+        interval: 1
+    database:
+        interval: 10
```

Comment:
One item left todo: should we put the database file name in the config yml file?